src/components/Table/Table.jsx

```diff
@@ -27,41 +27,79 @@ const cellProps = (
 
 function Table(props) {
   const { currentSort, fields, data, ariaLabelledBy } = props;
+  const [sortDirection, setSortDirection] = useState(currentSort?.order);
+  let sortLabel;
+  let rowData;
+
+  useEffect(() => {
+    rowData = sortData();
+    sortLabel = getSortLabel();
+  });
```

Comment:
I'm not sure why these are being assigned inside an effect. Can you explain why?

Reply:
When the `sortDirection` state is changed, then we re-sort the the data and set the `aria` sort label.

Actually, I can remove the `sortLabel` stuff.